python/ray/serve/tests/test_cli.py

```diff
@@ -667,6 +667,43 @@ def test_run_application(ray_start_stop):
     print("Kill successful! Deployment is not reachable over HTTP.")
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="File path incorrect on Windows.")
```

Comment:
Add a simple failure test case when you send ill-format yaml file.

Reply:
Added!